### PR TITLE
Enable verbose mode installation in CI/tox

### LINKED ISSUE #1530: Perform test installation in verbose mode
need to add to `tox.ini`
```
install_command =
    python -m pip install -v {opts} {packages}
``` 

and call `tox -vv`

_Originally posted by @themperek in https://github.com/cocotb/cocotb/pull/1425/files_

### PR DESCRIPTION
**Pull request by themperek**
Excluded on Windows since the log file is too long.

Resolves #1530

### CODE REVIEW

**Comment by ktbarrett** [comment]
Perhaps too verbose... :^)

File: .travis.yml [hunk 1]
```diff
@@ -36,8 +36,8 @@ jobs:
         - docker ps -a
       script:
         - echo 'Running cocotb tests with Python 3.5 ...' && echo -en 'travis_fold:start:cocotb_py35'
-        # - docker exec -it -e TRAVIS=true cocotb bash -lc 'pip3 install tox; cd /src; tox -e py35'
-        - docker exec -it -e TRAVIS=true cocotb bash -lc 'pip3 install importlib-resources==1.0.2 tox; cd /src; tox -e py35'  # remove once gh-1469's upstream bug is resolved
+        # - docker exec -it -e TRAVIS=true cocotb bash -lc 'pip3 install tox; cd /src; tox -vv -e py35'
+        - docker exec -it -e TRAVIS=true cocotb bash -lc 'pip3 install importlib-resources==1.0.2 tox; cd /src; tox -vv -e py35'  # remove once gh-1469's upstream bug is resolved
```
Comment (themperek) [inline .travis.yml]:
```suggestion
        # - docker exec -it -e TRAVIS=true cocotb bash -lc 'pip3 install tox; cd /src; tox -e py35'
        - docker exec -it -e TRAVIS=true cocotb bash -lc 'pip3 install importlib-resources==1.0.2 tox; cd /src; tox -e py35'  # remove once gh-1469's upstream bug is resolved
```

**Comment by eric-wieser** [comment]
I was hoping there was some way for us to get all the build_ext output without having to get pages of information about the package index.

**Comment by themperek** [comment]
> I was hoping there was some way for us to get all the build_ext output without having to get pages of information about the package index.

Can run `python setup.py build` before `tox`. Do not know another way at the moment.

**Comment by themperek** [comment]
I am closing since this produced a large amount not too useful output. If anyone finds this interesting please reopen/comment.

**Pull request closed without merging**